src/resources/dart-3-migration.md

```diff
@@ -596,6 +596,12 @@ analyzer:
   have been cleaned up [#50700](https://github.com/dart-lang/sdk/issues/50700).
 * The output of `dart format` changed a bit for
   [some code](https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#formatter).
+* On Windows the default location for the `PUB_CACHE` changed to
```

Comment:
This sounds like the change happened in 3.0. I thought it happened in 2.8, and it's just that 3.0 no longer supports the old location?

Reply:
Correct, I'll try to find better words :see_no_evil: